Source/WebKit/Shared/API/c/WKURL.h

```diff
@@ -35,6 +35,7 @@ extern "C" {
 WK_EXPORT WKTypeID WKURLGetTypeID(void);
 
 WK_EXPORT WKURLRef WKURLCreateWithUTF8CString(const char* string);
+WK_EXPORT WKURLRef WKURLCreateWithUTF8CStringWithLength(const char* string, size_t length);
```

Comment:
I think `CString` means null-terminated so this should be `WKURLCreateWithUTF8String`.

Reply:
I agree but I tried to follow the existing pattern in WKString:
```
WK_EXPORT WKStringRef WKStringCreateWithUTF8CString(const char* string);
WK_EXPORT WKStringRef WKStringCreateWithUTF8CStringWithLength(const char* string, size_t stringLength);
```

Reply:
But I'll go with `WKURLCreateWithUTF8String`

Reply:
I see: too bad about that pattern from `WKString`!